doc/sphinx/reference/dso_versioning_guide.md

```diff
@@ -184,8 +184,8 @@ Additionally, there's the complication of passing objects between different
 versions. For the moment, we can not bother trying to make this work.
 
 
-See also: 
+See also:
 ---------
 [Binutils manual](https://sourceware.org/binutils/docs/ld/VERSION.html)
 
-[Former glibc maintainer's DSO guide](http://www.akkadia.org/drepper/dsohowto.pdf)
```

Comment:
I don't really care, but is there a reason for preferring https for assets that don't benefit from encryption?

Reply:
The link check doesn't like it :)

Reply:
> I don't really care, but is there a reason for preferring https for assets that don't benefit from encryption?

Privacy.